locales/en-US.json

```diff
@@ -296,33 +298,33 @@
   "username": "Username",
   "usernameRequired": "Username is required",
   "validation": {
+    "bothPasswordsSame": "Both passwords must be the save",
     "mustBeUrl": "This field must be a valid URL",
-    "required": "This field is required",
-    "bothPasswordsSame": "Both passwords must be the save"
+    "required": "This field is required"
   },
   "version": "Version",
   "video": "Video",
   "videoTypes": "Video Types",
   "viewDetails": "View details",
   "vueClientVersion": "Vue client version",
   "wizard": {
+    "administratorAccount": "Administrator account",
+    "allowRemoteAccess": "Allow remote access to the server",
+    "completeError": "Unable to complete startup wizard.",
```

Comment:
Not for this PR, but I also think that we should put all the snackbar messages under a ``snackbar`` key.